app/src/main/java/org/glucosio/android/presenter/AddGlucosePresenter.java

```diff
@@ -112,7 +110,7 @@ private boolean createReading(String type, String notes, long oldId, Date finalD
     }
 
     public double convertToMmol(int mgDl) {
```

Comment:
Hmm.. inline ? how?? because other usages are also working like this. They should be all changed to inline! 

So would you recommend your thought?